contracts/lib/QuoteV3Auth/V3Struct.sol

```diff
@@ -41,6 +46,13 @@ library V3Struct {
         bytes certData;
     }
 
+    struct ParsedCertificationData {
```

Comment:
Also redundant, but let's store certType as a uint16, and certDataSize as a uint32.

For the cert data array however, I do not recommend storing them as a fixed length at 3. As seen from the Intel source code at https://github.com/intel/SGX-TDX-DCAP-QuoteVerificationLibrary/blob/16b7291a7a86e486fdfcf1dfb4be885c0cc00b4e/Src/AttestationLibrary/src/QuoteVerification/QuoteStructures.h#L135-L141, they are stored as a vector instead.

This essentially provides us with the flexibility to store a cert chain of various size, if Int